{"name": "test_messgaeConfirm", "status": "broken", "statusDetails": {"message": "NameError: name 'driver' is not defined", "trace": "@allure.story('密碼輸入/再次輸入')\n    @pytest.mark.run(order=6)\n    def test_messgaeConfirm():\n>           WebDriverWait(driver, 10).until(EC.presence_of_element_located((By.XPATH, \"/html/body/div[1]/div/div/main/div/div[2]/div/div/div/form/label[1]/div/div[1]/div[1]/input\")))\nE           NameError: name 'driver' is not defined\n\nSSO_login_normal_test.py:94: NameError"}, "start": 1669948605937, "stop": 1669948605937, "uuid": "2ea85d22-94bb-4d31-98a3-6b3f00bf30df", "historyId": "ce881bc0de7d97a6b86820599b476587", "testCaseId": "730a35a7d30e9cbb3947165c1b0a2d9a", "fullName": "SSO_login_normal_test#test_messgaeConfirm", "labels": [{"name": "story", "value": "密碼輸入/再次輸入"}, {"name": "tag", "value": "run(order=6)"}, {"name": "suite", "value": "SSO_login_normal_test"}, {"name": "host", "value": "C010700042"}, {"name": "thread", "value": "1664-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "SSO_login_normal_test"}]}